exquery-restxq-specification/restxq-1.0-specification.html

```diff
@@ -152,16 +152,16 @@
       <a href="http://exquery.org/" title="EXQuery: Extensions for EXQuery">EXQuery project</a>.
     </section>
     <section id='abstract'>
-      Whilst XQuery [[!XQUERY]] was originally envisaged and designed as a query language for XML,
+      Whilst XQuery was originally envisaged and designed as a query language for XML,
       it has been adopted by many as a language for application development. This specification
-      describes a set of XQuery Annotations [[!XQUERY-30]] and a small set of functions to enable XQuery
+      describes a set of XQuery Annotations and a small set of functions to enable XQuery
```

Comment:
I think this should be `of XQuery Annotations (see [[XQUERY-30]]) and`

Reply:
I think also two more sections need to be changed from normative to informative references:
1. `namespaces (see [[!XML-NAMES]]) and referenced` (line ~234) -> `namespaces (see [[XML-NAMES]]) and referenced`.
2. `RESTXQ is heavily influenced by [[!JAX-RS]]` (line ~257) -> `RESTXQ is heavily influenced by JAX-RS (see [[JAX-RS]])`.